@ndlib/gatsby-theme-marble/gatsby-node.js

```diff
@@ -28,20 +30,31 @@ exports.sourceNodes = ({ actions }) => {
     default: String
     thumbnail: String
   }
+  type MarbleIiifImage implements Node @dontInfer {
+    id: String!
+    marbleId: String!
+    service: String
+    default: String
+    thumbnail: String
+    title: String
+    collection: MarbleItem
+    parent: MarbleItem
+    sequence: Int
+  }
   type MarbleItem implements Node @dontInfer {
     id: String!
     marbleId: String!
     slug: String!
     display: String
     title: String!
     description: String
+    sequence: Int
     iiifUri: String
     partiallyDigitized: Boolean
     metadata: [metadataData]
-    image: imageData
-    allImages: [imageData]
     copyrightRestricted: Boolean
-    seeAlso: [String]
+    childrenMarbleItem: [MarbleItem]
+    childrenMarbleIiifImage: [MarbleItem]
```

Comment:
Should this one be?
```
childrenMarbleIiifImage: [MarbleIiifImage]
```

Reply:
oh yes.  we still get the warning message about these though.  it says that if you have defined it you can ignore the warning so realistically i am not 100% sure i addressed it but i think it did.